src/kemal/helpers/exceptions.cr

```diff
@@ -13,7 +13,10 @@ module Kemal::Exceptions
   end
 
   class CustomException < Exception
-    def initialize(context : HTTP::Server::Context)
+    def initialize(context : HTTP::Server::Context, message : String = "")
+      @context = context
+      @message = message
```

Comment:
To be correct this IMO should call `super message` instead of directly setting the @ivar.
Also, `message` should be nil-able, as in `Exception.new` constructor.